QuickTests/FunctionalTests/DescribeTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+@testable import Quick
+import Nimble
+
+class DescribeTests: XCTestCase {
+    
+    func testDescribeErrorsIfExampleGroupsAreNotInstantiated() {
+        
+        // fiddling with global state in this test to test a singleton, what could go wrong?
+        let exampleGroup = World.sharedWorld.currentExampleGroup
+        World.sharedWorld.currentExampleGroup = nil
```

Comment:
Agreed... Would it be possible to write a test that encounters the same error a user would? That is, a test that uses a `describe()` outside of `QuickSpec.spec()`?

Something like:

```swift
import Quick

var isRunningTest = false

class DescribeThrowsTests: XCTestCase {
    func testUsingADescribeOutsideOfASpecThrows() {
        if isRunningTest {
            describe("This will cause an exception to be thrown") {}
        }
    }
}

class DescribeThrowsSpec: QuickSpec {
    override func spec() {
        expect { qck_runSpec(DescribeThrowsTests.classForCoder()) }
            .to(raiseException())
    }
}
```

This may not work because of how Quick initializes its example tree, but it might be worth a shot!